book/src/keymap.md

```diff
@@ -156,13 +156,14 @@ Jumps to various locations.
 | `l`   | Go to the end of the line                        | `goto_line_end`            |
 | `s`   | Go to first non-whitespace character of the line | `goto_first_nonwhitespace` |
 | `t`   | Go to the top of the screen                      | `goto_window_top`          |
-| `m`   | Go to the middle of the screen                   | `goto_window_middle`       |
+| `c`   | Go to the middle of the screen                   | `goto_window_middle`       |
```

Comment:
Maybe we should rename it to `goto_window_center` to be consistent with the keybind ?

Reply:
Sure.